js/scripts.js

```diff
@@ -90,12 +90,12 @@ var MoviePoster = React.createClass({
 
   render: function () {
     return (
-      React.createElement('img', {src: this.props.moviePoster})
+      React.createElement('img''alt'="poster", {src: this.props.moviePoster})
     );
   }
 });
 
-var app = React.createClass({
+var app = React.CreateClass({
```

Comment:
same as above, `createClass` is a method - is should be name as in docs

`app` is an variable, variable name should start with big letter